test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -137,7 +138,7 @@ func SetupNVIDIAGPUNode(f *framework.Framework, setupResourceGatherer bool) *fra
 
 	framework.Logf("Using %v", dsYamlURL)
 	// Creates the DaemonSet that installs Nvidia Drivers.
-	ds, err := framework.DsFromManifest(dsYamlURL)
+	ds, err := e2emanifest.DaemonSetFromManifest(dsYamlURL)
```

Comment:
Can we rename `DaemonSetFromManifest` to `DaemonSetFromURL`?

Reply:
sure . thanks your review.

Reply:
ha! this has a better ring to it! :+1: